devtools/maven/pom.xml

```diff
@@ -111,6 +111,12 @@
             <groupId>org.twdata.maven</groupId>
             <artifactId>mojo-executor</artifactId>
             <version>2.3.0</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>slf4j-simple</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
I'm not sure we have JBoss Logging properly set up in the Maven stuff? Just checking.

Reply:
Added slf4j-jboss-logging (the slf4j JBoss Logging impl)